Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckPathLengthCommand.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.IO;
+using PatchKit.Api.Models.Main;
+using PatchKit.Unity.Patcher.AppUpdater.Status;
+using PatchKit.Unity.Patcher.Cancellation;
+using PatchKit.Unity.Patcher.Debug;
+
+
+namespace PatchKit.Unity.Patcher.AppUpdater.Commands
+{
+    public class CheckPathLengthCommand : ICheckPathLengthCommand
```

Comment:
I'd suggest using ContentSummary for both cases (content installation & diff update).

Reply:
done